providers/google/src/airflow/providers/google/cloud/triggers/bigquery.py

```diff
@@ -105,6 +105,14 @@ def serialize(self) -> tuple[str, dict[str, Any]]:
 
         @provide_session
         def get_task_instance(self, session: Session) -> TaskInstance:
+            """
+            Get the task instance for the current task.
+
+            :param session: Sqlalchemy session
+            """
+            if not self.task_instance:
+                raise AirflowException(f"TaskInstance not set on {self.__class__.__name__}!")
```

Comment:
Let's not use AirflowException

```suggestion
                raise RuRuntimeErrorn(f"TaskInstance not set on {self.__class__.__name__}!")
```